katka/migrations/0003_add_slug_with_default.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.1.5 on 2019-02-07 06:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('katka', '0002_project'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='project',
+            name='slug',
+            field=models.SlugField(default='NONE', max_length=10),
```

Comment:
There are three migrations in this one commit:
- Added project
- Added slug to both team and project
- Removed default from slug

The reason why this is not squashed is:
- because you can't add a non-null without a default (because existing databases need a value), but you can remove it later to have no default.
- I think adding the slug separate to team and product shows better than something changed in the team model as well (but honestly, I felt a bit lazy to squash these two).

Reply:
Since I had to fix linting errors, I went back and looked again and found a way to do it in one migration. I think this is better and I should have done this in the first place.